algebra/primefield/src/test.rs

```diff
@@ -1,14 +1,18 @@
-use crate::{Parameters, PrimeField, UInt};
+use crate::{u256::U256, Parameters, PrimeField};
 use quickcheck::{Arbitrary, Gen};
 
-impl<U, P> Arbitrary for PrimeField<P>
+impl<P> Arbitrary for PrimeField<P>
```

Comment:
Why can we not have `Arbitrary` be generic over the backing uint implementation?

Reply:
We could, but i don't think it really matters, because this is going to be deleted once prime-field migrates to proptest